markdown/developers/adding_modules.md

```diff
@@ -515,6 +515,10 @@ If a new test dataset is added to [`tests/config/test_data.config`](https://gith
 
 For example: the nf-core/test-datasets file `genomics/sarscov2/genome/genome.fasta` labelled as `genome_fasta`, or `genomics/sarscov2/genome/genome.fasta.fai` as `genome_fasta_fai`.
 
+### Using a stub test when required test data is too big
+
+If the module absolute cannot run using tiny test data, there is a possibility to add `-stub-run` to the test.yml. In this case it is required to test the module using larger scale data and document how this is done. In addition, an extra script-block labeled `stub:` must be added, and this block must `touch` all expected output files and the `versions.yml`. An example is found in the [ascat module](https://github.com/nf-core/modules/blob/73aaecbc3ae62a124e1148b68b1a69dcfd9db0ed/modules/ascat/main.nf#L137-L152). In the `test.yml` the `-stub-run` argument and the md5sum are manually entered as empty-file md5sums (`d41d8cd98f00b204e9800998ecf8427e`) for each of the files that are added in the stub-block. This causes the stub-code block to be activated ([example](https://github.com/nf-core/modules/blob/73aaecbc3ae62a124e1148b68b1a69dcfd9db0ed/tests/modules/ascat/test.yml)).
```

Comment:
Also maybe we could include the command to run a pipeline using the stub run option?

Reply:
This should be included in the `test.yml`

Reply:
Ok, an example is [now shown](https://github.com/nf-core/nf-co.re/pull/1079/files#diff-cdd07964dfe75f43fdf43da7c94ddd2ade366292dcfee33af3819e5d6249c3f1R523). It's also in test.yml @Emiller88 thanks.